internal/ingest/ingest_test.go

```diff
@@ -199,7 +199,9 @@ func TestFailDuringResync(t *testing.T) {
 			{ChunkCount: 1, EntriesPerChunk: 1, EntriesSeed: 2},
 		}}.Build(t, te.publisherLinkSys, te.publisherPriv)
 
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
I personally feel like the test should just timeout here and we shouldn't worry about setting timeouts everywhere.

Reply:
Previously it was after 10m (before fix was made), which was an annoying time to wait for failure, so I added the timeout.